proposals/description/cr_oracle_cleanup.ts

```diff
@@ -0,0 +1,385 @@
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [addresses.d3poolCurvePCVDeposit, addresses.frax, addresses.tribalCouncilSafe, '10000'],
+      description: 'Withdraw all FRAX on d3 deposit to TC Multisig where it can be sold'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [
+        addresses.d3poolCurvePCVDeposit,
+        addresses.alusd,
+        addresses.tribalCouncilSafe,
+        '10000'
+      ],
+      description: 'Withdraw all alUSD on d3 deposit to TC Multisig where it can be sold'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [addresses.compoundEthPCVDeposit, addresses.comp, addresses.tribalCouncilSafe, '10000'],
+      description: 'Withdraw all COMP from Compound ETH deposit to TC Multisig where it can be sold'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [addresses.compoundDaiPCVDeposit, addresses.comp, addresses.tribalCouncilSafe, '10000'],
+      description: 'Withdraw all COMP from Compound DAI deposit to TC Multisig where it can be sold'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [addresses.aaveEthPCVDeposit, addresses.stAAVE, addresses.tribalCouncilSafe, '10000'],
+      description: 'Withdraw all stkAAVE from Aave ETH deposit to TC Multisig where it can be sold'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [addresses.d3poolConvexPCVDeposit, addresses.crv, addresses.tribalCouncilSafe, '10000'],
+      description: 'Withdraw all CRV from Convex deposit to TC Multisig where it can be sold'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [addresses.d3poolConvexPCVDeposit, addresses.cvx, addresses.tribalCouncilSafe, '10000'],
+      description: 'Withdraw all CVX from Convex deposit to TC Multisig where it can be sold'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [addresses.bammDeposit, addresses.lqty, addresses.tribalCouncilTimelock, '10000'],
+      description: 'Withdraw all LQTY from B.AMM deposit to TC Timelock'
```

Comment:
We don't plan to sell the LQTY and so aren't sending to the Safe?

Reply:
yes that's the idea